### PR TITLE
WIP- Moved Plan and Task Domain Classes to OpenSRP Plan evaluator

### PR DESCRIPTION
**Pull request by rehammuzzamil**
Issue # 10
Dependent PR : https://github.com/OpenSRP/opensrp-plan-evaluator/pull/12

### CODE REVIEW

**Comment by githengi** [comment]
@rehammuzzamil move `org.opensrp.domain.PhysicalLocationTest` and `org.opensrp.domain.TaskTest` to planEvaulator library

File: pom.xml [hunk 1]
```diff
@@ -24,6 +24,7 @@
 		<opensrp.common.version>2.0.0-SNAPSHOT</opensrp.common.version>
 		<opensrp.api.version>1.0.1</opensrp.api.version>
 		<opensrp.interface.version>2.0.1-SNAPSHOT</opensrp.interface.version>
+		<opensrp.plan.evaluator.version>0.0.2-SNAPSHOT</opensrp.plan.evaluator.version>
```
Comment (githengi) [inline pom.xml:27]: change to version 0.0.3

**Comment by githengi** [comment]
@rehammuzzamil move `org.opensrp.domain.PhysicalLocationTest` and `org.opensrp.domain.TaskTest` to planEvaulator library

File: pom.xml (hunk 1 above)
Comment (rehammuzzamil) [reply pom.xml]: Done

**Comment by rehammuzzamil** [comment]
> @rehammuzzamil move `org.opensrp.domain.PhysicalLocationTest` and `org.opensrp.domain.TaskTest` to planEvaulator library

Removed TaskTest.java class, I have not removed PhysicalLocationTest as it is dependent on PhysicalLocationServiceTest, also domain class of PhysicalLocation is not moved into plan-evaluator-library in this PR, it is under the scope of https://github.com/OpenSRP/opensrp-plan-evaluator/issues/16  @githengi

**Pull request merged**